metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -712,7 +714,9 @@ def loop_tables(  # noqa: C901
                             data_reader,
                         )
                     except Exception as e:
-                        self.warn(logger, f"{schema}.{table}", f"Ingestion error: {e}")
+                        self.warn(
```

Comment:
we should have two lines - one call to `self.warn(...)` with just the `{e}`, and one call to `logger.debug(..., exc_info=True)` to show the stack trace

we don't want stack traces in the ingestion report, only in the logs

Reply:
Got it, updated it.